### DISCUSSION TITLE
Examples for usage?

### DISCUSSION THREAD

**Discussion by Lioncat2002** [DanielChappuis/reactphysics3d#332] [Q&A]
Hello!
Are there any examples on how to use the various components?
I found the basic example for a falling rigidbody. However I want to add collisions to it.
Would be great if there were a few examples which can show how to achieve it.

I am pretty new to OpenGL and C++ in general. So struggling a bit 😅

**Comment by DanielChappuis** [comment]
Welcome aboard. You should first check the documentation of the library [here](https://reactphysics3d.com/documentation.html).

Secondly, you can check all the examples in the [testbed](https://github.com/DanielChappuis/reactphysics3d/tree/master/testbed) application (that you can find in the same repository). This demo application uses OpenGL to display several examples of physics simulation and you can also see their source code.

Happy coding !

**Reply by Lioncat2002** [reply]
Ahh Thank you so much!
